### ISSUE TITLE
When admin user deletes an user account, MeshCentral "crashed"

### ISSUE THREAD

**Issue by hunter-nl** [Ylianst/MeshCentral#282]
This is the error:
```
-------- 6/7/2019, 11:49:00 AM ---- 0.3.6-c --------

/opt/meshcentral/node_modules/meshcentral/meshuser.js:57
        if (parent.wssessions[obj.user._id]) {
                                       ^

TypeError: Cannot read property '_id' of undefined
    at Object.obj.close (/opt/meshcentral/node_modules/meshcentral/meshuser.js:57:40)
    at WebSocket.<anonymous> (/opt/meshcentral/node_modules/meshcentral/meshuser.js:279:49)
    at WebSocket.emit (events.js:205:15)
    at WebSocket.emitClose (/opt/meshcentral/node_modules/express-ws/node_modules/ws/lib/websocket.js:172:10)
    at Socket.socketOnClose (/opt/meshcentral/node_modules/express-ws/node_modules/ws/lib/websocket.js:781:15)
    at Socket.emit (events.js:200:13)
    at TCP.<anonymous> (net.js:586:12)
```

**Comment by Ylianst** [comment]
Thanks for the report. Fixed it, will be in MeshCentral v0.3.6-f when I publish it.

**Comment by hunter-nl** [comment]
Yes, fixed. Thanks!
